src/transcribo_backend/services/whisper_service.py

```
@@ -83,129 +88,119 @@ async def transcribe_get_task_result(task_id: str) -> TranscriptionResponse:
-
-    # Send the request
-    async with (
-        aiohttp.ClientSession() as session,
-        session.post(url, data=form_data) as response,
-    ):
+        return TaskStatus(**response.json())
+
+    async def transcribe_submit_task(
+        self,
+        audio_data: bytes,
+        file_format: str,
```

Ruff (0.14.11)

Unused method argument: `file_format`

(ARG002)